bin/rspecq

```diff
@@ -110,6 +117,7 @@ else
     redis_host: opts[:redis_host]
   )
 
+  worker.fail_fast = opts[:fail_fast]
```

Comment:
Minor but let's move this below `max_requeues`, so that the list reads more naturally (e.g. `files_or_dirs_to_run` is the most important/core option here).